### ISSUE TITLE
Faction Area feature ? 

### ISSUE THREAD

**Issue by ghost** [BlockHorizons/FactionsPE#88]
If you found a faction and claim your area you should maybe be able to see where the borders of the landscape are, so with red carpets, depending on the extension, the border should be displayed 

Please add the feature Please

**Comment by Kris-Driv** [comment]
I'll make something similar to your suggestion. But currently you can use `/faction map` and `/faction seechunk`

**Comment by ghost** [comment]
No, I meant that you can not see the claimed area through a map or something but as an orientation where a faction begins and in Conquest games where the borders are shown in colors where a faction begins and by expanding the region the border is simply moved and enlarged

**Comment by ghost** [comment]
I search a Picture for example for you

**Comment by DragonfireHD** [comment]
![Uploading PSX_20201228_205957.jpg…]()
I mean that

**Comment by DragonfireHD** [comment]
![Uploading PSX_20201228_205957.jpg…]()

**Comment by DragonfireHD** [comment]
![Uploading PSX_20201228_205957.jpg…]()

**Comment by DragonfireHD** [comment]
![Uploading PSX_20201228_210614.jpg…]()

**Comment by ghost** [comment]
![Uploading InShot_20201228_210734653.jpg…]()

**Comment by ghost** [comment]
Ok Github is broke for Pictures

**Comment by ghost** [comment]
I send the link from Picture and can see the borders from Factions i mean the Border to see the border on the ground with colors with carped 


Link 
https://images.app.goo.gl/Lr8bE8FFYquVKfVHA

**Comment by Kris-Driv** [comment]
This is good suggestion. I'll implement this at some point. Thanks!

**Comment by ghost** [comment]
> This is good suggestion. I'll implement this at some point. Thanks!

hey dude, suggestion to fix poggit lint and formatting the codes to make it neat

**Comment by Kris-Driv** [comment]
> > This is good suggestion. I'll implement this at some point. Thanks!
> 
> hey dude, suggestion to fix poggit lint and formatting the codes to make it neat

I agree that lint should be fixed. But code is formatted just fine. Perhaps you can point out at some example?
